promql/promqltest/testdata/subquery.test

```diff
@@ -102,15 +102,15 @@ eval instant at 80s rate(metric[1m])
 
 # No extrapolation, [2@20, 144@80]: (144 - 2) / 60
 eval instant at 80s rate(metric[1m:10s])
-  {} 2.366666667
+  {} 2.4
```

Comment:
The test intends to create 2.366666667 as a result, as per the formula above. So we need to change the ranges in the subquery accordingly. I suggest to use `rate(metric[1m5s:10s])`